<launch>

 <param name="/use_sim_time" value="false"/> 

 <rosparam file="$(find avt_341)/config/waypoints_na.yaml" />
  
 <include ns="fed_alpha" file="$(find avt_341)/launch/lead_vehicle.launch"/>

 <include ns="fed_alpha_1" file="$(find avt_341)/launch/lead_vehicle.launch"/>

</launch>
